CDP4IME/CDP4IMEBootstrapper.cs

```diff
@@ -98,17 +96,12 @@ protected override void InitializeShell()
             var shell = (Shell)this.Shell;
             var dialogNavigationService = this.Container.GetExportedValue<IDialogNavigationService>();
             var dockViewModel = this.Container.GetExportedValue<DockLayoutViewModel>();
+            var ribbonBuilder = this.Container.GetExportedValue<IRibbonContentBuilder>();
 
+            //'Make do' solution until the bootstrapper is replaced. Currently injecting the builder into the Shell view is rejected by the composition.
```

Comment:
can you add a TODO: here so VS2019 will find it and also add a reference to a GitHub ticket?